linux_os/guide/system/software/system-tools/package_scap-security-guide_installed/rule.yml

```diff
@@ -32,6 +32,10 @@ ocil_clause: 'the package is not installed'
 
 ocil: '{{{ ocil_package(package="scap-security-guide") }}}'
 
+fixtext: "{{{ fixtext_package_installed("scap-security-guide") }}}"
```

Comment:
This also generates weird texts:


To install the Red Hat Enterprise Linux 9 package scap-security-guide run the following command: The scap-security-guide package can be installed with the following command: $ sudo dnf install scap-security-guide